build/Common.prod.props

```diff
@@ -77,7 +77,9 @@
     <!-- Note: This selects the correct PublicApiAnalyzers files based on $(ExposeExperimentalFeatures) -->
     <ItemGroup>
       <AdditionalFiles Include=".publicApi\Stable\$(TargetFramework)\PublicAPI.*.txt" />
```

Comment:
Is this still needed?

Reply:
Ya it is. After this PR API project still has `\Stable\net462\PublicAPI.[Shipped|Unshipped].txt` which needs to be included. For projects that don't have TFM-specific files it just no-ops.